src/package.json

```diff
@@ -37,9 +43,14 @@
     "winston": "^3.2.1"
   },
   "devDependencies": {
+    "eslint": "^6.8.0",
+    "eslint-config-airbnb-base": "^14.1.0",
+    "eslint-plugin-import": "^2.20.2",
+    "husky": "^4.2.3",
     "jest": "^25.2.4",
     "nodemon": "^2.0.2",
     "supertest": "^4.0.2",
+    "uri-js": "^4.2.2",
```

Comment:
Where was this used?

Reply:
It's from eslint. It seems when I installed eslint it didn't include uri-js for some reason.

Reply:
Okay. Just leave it as it is. It's a dev dependency anyway. 👍